tiers/checks.go

```diff
@@ -33,6 +33,10 @@ func NewChecks(tier string) (TierChecks, error) {
 	case "basic":
 		return &basicTierChecks{}, nil
 
+	case "basic-no-deactivation":
+		// we want the basic-no-deactivation tier to have the same resources as the basic tier with the only difference being auto deactivation disabled
```

Comment:
@alexeykazakov is it safe to assume we want the 'basic-no-deactivation' tier to be consistent with the 'basic' tier in terms of the resources so using the same checks is what we want?

Reply:
yes

Reply:
this is why it's called `basic-no-deactivation` ;)

Reply:
btw, I'm a bit surprised that we verify everything from the tier, but not the deactivation information. 
Shouldn't we verify somewhere also the field `DeactivationTimeoutDays` of the created NSTemplateTiers?

Reply:
Well.. the current tier checks verify namespace and cluster resources created by the tiers only.
Deactivation configuration is part of the tier resource itself and tested indirectly in our deactivation tests.
But I guess we should add a new tier check type. Currently we have only `namespaceObjectsCheck` and `clusterObjectsCheck`. Let's add something like `tierObjectsCheck` which will verify the Tier resource itself so we can make sure only `basic` tier has deactivation enabled and the rest of the tiers do not have it.

Reply:
Thanks for the good point, I've added `tierObjectCheck` to validate the `DeactivationTimeoutDays`